x/time_test.go

```diff
@@ -8,11 +8,11 @@ import (
 )
 
 func TestRandomDelay(t *testing.T) {
-	base := time.Millisecond * 20
-	deviation := time.Millisecond * 5
-	start := time.Now()
-	RandomDelay(base, deviation)
-	delay := time.Since(start)
-	require.LessOrEqual(t, delay, base+deviation)
-	require.GreaterOrEqual(t, delay, base-deviation)
+	base := time.Millisecond * 2
+	deviation := time.Millisecond
+	for i := 0; i < 100; i++ {
```

Comment:
@mattbonnell I was getting flaky tests and changed this so that it returns the duration instead of sleeping which makes it harder to debug. It's now failing consistently:

```
=== RUN   TestRandomDelay
    assertion_compare.go:342: 
        	Error Trace:	time_test.go:15
        	Error:      	"3.219555ms" is not less than or equal to "3ms"
        	Test:       	TestRandomDelay
        	Messages:   	[]
```

Can you please take a look and fix this? Thank you!

Reply:
yep let me take a look!

Reply:
okay no, seems to be an issue with the math :) will fix this today.

Reply:
fix PR https://github.com/ory/kratos/pull/1131